source/educators/references/resources_for_course_teams.rst

```diff
@@ -67,29 +67,25 @@ a PDF version, select **v: latest** at the lower right of the page, then select
 **PDF**.
 
 .. note:: If you use the Safari browser, be aware that it does not support the
- search feature for the HTML versions of the edX guides. This is a known
+ search feature for the HTML versions of the Open edX guides. This is a known
  limitation.
 
-======
+*****
 Email
-======
+*****
 
 To receive and share information by email, course team members can:
 
-* Subscribe to announcements and other new topics in the edX Partner
+* Subscribe to announcements and other new topics in the Open edX Partner
   Portal or the Open edX Portal. For information about how to subscribe, see
   the :doc:`Release Announcements through the Open edX Portal <../../community/how-tos/receive_announcements_by_email>`
 
 * Join the `openedx-studio`_ Google group to ask questions and participate in
-  discussions with peers at other edX partner organizations and edX staffers.
+  discussions with peers at other Open edX partner organizations and Open edX staffers.
 
-====================
+*******************
 Wikis and Web Sites
-====================
+*******************
 
-The edX product team maintains public product roadmaps on :ref:`the Open edX
-Portal<The Open edX Portal>` and :ref:`the edX Partner Portal<The edX Partner
-Portal>`.
+The Open edX product team maintains public product roadmaps on `the Open edX Portal <https://openedx.org/>` and `the Open edX Commnity <https://openedx.org/community/>`_.
```

Comment:
This is no longer accurate. Please reference the public roadmap at https://github.com/orgs/openedx/projects/4 and how to get involved with defining the roadmap by joining the Product Working Group https://openedx.atlassian.net/wiki/spaces/OEPM/overview

Reply:
done